src/main/java/seedu/address/model/timeframe/TimeFrame.java

```diff
@@ -47,13 +47,25 @@ public LocalDateTime getEndTime() {
         return to;
     }
 
+    /**
+     * Set end time of the timeframe to toDate
+     * @param td
+     */
+    public void setEndTime(LocalDateTime toDate) {
+        to = toDate;
+    }
+
     /**
      * Gets the number of days in the timme frame
      */
     public long getDays() {
         return from.until(to, DAYS) + 1;
     }
 
```

Comment:
Would it be better to add in java docs here? A bit confused how this method works in `addExtendRoomCost(RoomCost roomCost, Stay stay, LocalDateTime fromDate)` in Bill.java

Reply:
added